Dockerfile.arm

```diff
@@ -0,0 +1,13 @@
+FROM resin/armel-debian:jessie
```

Comment:
For now we will not build an installer for arm, only for amd64 and arm64. The one for arm64 is based on aarch64/debian:jessie and it comes with a limitation: it does not install grub.